.devcontainer/postCreateCommand.sh

```diff
@@ -20,3 +20,4 @@ mkdir $HOME/.kube
 echo "source <(kubectl completion bash)" >> $HOME/.bashrc
 echo "alias k=kubectl" >> $HOME/.bashrc
 echo "complete -F __start_kubectl k" >> $HOME/.bashrc
+docker network create -d=bridge -o com.docker.network.bridge.enable_ip_masquerade=true -o com.docker.network.driver.mtu=1500 --subnet fc00:f853:ccd:e793::/64 kind
```

Comment:
wow what the hell, I'm surprised by this.

Reply:
Is this subnet something known ahead of time?

Reply:
Yeah it's known in codespaces. Definitely not a permanent fix. We need to upgrade kind once a new one is released.

Reply:
I was curious if it could be looked up by a script function?

Reply:
Not too sure tbh. If you have another approach I'm definitely interested.